<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rgbFilterDialog</class>
 <widget class="QDialog" name="rgbFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_Component">
         <property name="text">
          <string>Component</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Value">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_Component">
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_Value">
         <property name="suffix">
          <string/>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
